@stellar/design-system-website/docs/components/buttons/button.mdx

```diff
@@ -1,8 +1,8 @@
 ---
-slug: /buttons
+slug: /button
 ---
 
-# Buttons
+# Button
```

Comment:
Using a component name for slug and title to make it easier to remember how to link to other components using a tag.